docs/docs/architecture.md

```diff
@@ -1,143 +1,148 @@
-[Mail](#mail) section below for further discussion of this.
-
-Whilst the high level setup has just those three entities, real deployments have more:
-
-1. Intel
-2. Optionally, a cloud provider
-3. Optionally, an auditor
-
-**Intel.** Intel's involvement in a deployed architecture is limited to providing the CPU hardware and running some servers.
-These servers provision the host with certificates guaranteeing authenticity of the chip (once, on initial setup)
-and provide the client with an assessment of the security of the host given a remote attestation (whenever the
-client requests it). If Intel's servers become unreachable by the clients that's not a problem, it just means the
-security of the host machine may degrade without the clients realising.
-
-For instance if the host's SGX software stack is out of date and has known vulnerabilities, or if the BIOS configuration is
-not correct, this will be reported to the client by Intel's servers as part of verifying and processing the data inside
-the remote attestation provided by the host.
-
-**Cloud provider.** A cloud provider needs to support SGX for Conclave to be usable. They may operate their own
-provisioning servers that take over from Intel's. You can find the instructions to use Conclave on Microsoft Azure
-[here](machine-setup.md).
-
-**Auditor.** In the pure enclave-oriented model, the user is responsible for understanding what the enclave does before
-using it by reading the enclave's source code. If the user doesn't do this then enclaves have no point. In practice the
-user may wish to outsource this auditing to someone else.
+Clients need to be sure that they are communicating with a real enclave. They cannot rely on the host to _assert_ 
+that this is the case because the host could be malicious. Instead, Conclave uses a process called
+*remote attestation*.
+
+First, clients need to obtain an
+[`EnclaveInstanceInfo`](api/-conclave%20-core/com.r3.conclave.common/-enclave-instance-info/index.html) object. You 
+can download this object on-demand from the host, or it could be published somewhere. 
+
+The `EnclaveInstanceInfo` object encapsulates a remote attestation. The remote attestation includes the following 
+information:
+
+* The hash of the code inside the enclave.
+* The hash of the key of any party who has signed that code.
+* The security status of the machine on which the enclave is running.
+* A public key whose private counterpart is known only to the enclave.
+
+The remote attestation encapsulated within the `EnclaveInstanceInfo` object is signed by Intel. So the client can have 
+confidence about the enclave even though they obtain the `EnclaveInstanceInfo` from an untrusted host.
+
+The client tests the
+[`EnclaveInstanceInfo`](api/-conclave%20-core/com.r3.conclave.common/-enclave-instance-info/index.html) against a
+set of [constraints](constraints.md). Constraints are represented by an
```

Comment:
Should be bold the world "constraints" here, or italics, because a new and important concept has just been introduced. Or perhaps this needs its own sub-heading?

Reply:
Changed to italics.